airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/RedshiftDestination.java

```diff
@@ -42,7 +42,7 @@ public static DestinationType getTypeFromConfig(final JsonNode config) {
   public static Map<DestinationType, Destination> getTypeToDestination() {
     return Map.of(
         DestinationType.INSERT_WITH_SUPER_TMP_TYPE, new RedshiftInsertDestination(RedshiftDataTmpTableMode.SUPER),
-        DestinationType.COPY_S3_WITH_SUPER_TMP_TYPE, new RedshiftCopyS3Destination(RedshiftDataTmpTableMode.SUPER));
```

Comment:
After this change RedshiftCopyS3Destination is not going to be used anywhere and should be deleted.

Reply:
> After this change RedshiftCopyS3Destination is not going to be used anywhere and should be deleted.

deleted RedshiftCopyS3Destination